scripts/main.js

```diff
@@ -39,16 +39,24 @@ Hooks.once('socketlib.ready', () => {
  */
 Hooks.once('ready', async () => {
   const settings = new Settings();
-
-  if (!settings.customEffectsItemId) {
+  const effectHelpers = new EffectHelpers();
+  let { customEffectsItemId } = settings;
+  if (!customEffectsItemId) {
     const item = await CONFIG.Item.documentClass.create({
       name: 'Custom Convenient Effects',
       img: 'modules/dfreds-convenient-effects/images/magic-palm.svg',
       type: 'consumable',
     });
 
-    await settings.setCustomEffectsItemId(item.id);
+    customEffectsItemId = await settings.setCustomEffectsItemId(item.id);

```

Comment:
Assigning this doesn't need to be done since you never use it again

Reply:
Indeed